README.md

```diff
@@ -65,6 +65,18 @@ The app should be running on port 5000. You should see output similar to the fol
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+## Authentication
```

Comment:
There's no note here about encryption, but given you completed the much bigger task I'll trust that's been done!